cmd/web3/main.go

```diff
@@ -205,6 +205,30 @@ func main() {
 				GetAddressDetails(ctx, network, c.Args().First(), privateKey, true, contractAddress)
 			},
 		},
+		{
+			Name:  "increasegas",
+			Usage: "Increase gas for a transaction. Useful if a tx is taking too long and you want it to go faster.",
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:        "private-key, pk",
+					Usage:       "The private key",
+					EnvVar:      pkVarName,
+					Destination: &privateKey,
+					Required:    true},
+				cli.StringFlag{
+					Name:     "tx",
+					Usage:    "The transaction hash of the pending transaction.",
+					Required: true},
+				cli.IntFlag{
+					Name:  "amount",
+					Usage: "The amount in GWEI to increase the price. 1 would add 1 more GWEI. (default: 1)",
+					Value: 1,
```

Comment:
Is `gwei` only too restrictive? We have [`web3.ParseAmount`](https://godoc.org/github.com/gochain/web3#ParseAmount) already that could drop in, but that doesn't actually handle all the units. Maybe `gwei` relative makes sense, but we could open up to parsing a decimal.

I wonder if an alternative percentage based flag could be useful too (`--pct 10`,`--pct 200`). 

Both of these could be follow ups.

Reply:
Pushed a decimal parsing version.

Reply:
Not sure what the decimal parsing would do, is it even aloud?  I've never seen gas be less than a gwei. Did you test that it works?

Reply:
Percent increase makes sense. I usually double it, so a 2x bump would be a nice flag to have.

Reply:
I thought about something like that too. Maybe an alternative flag, or a % suffix on the amount.